canary_config.example.py

```diff
@@ -15,3 +15,7 @@
     type="robot-location-secret",
     payload="put-the-secret-here")
 address = "address-for-robot.viam.cloud"
+
+# Who watches the watchmen? Most canaries should leave this as None, but we need at least one to
+# set it to whichever board is making sure all the others are online (e.g., "pi5canary").
+ssh_monitor = None
```

Comment:
the upboard or beaglebone are relatively reliable, I think one of those could monitor pi5

Reply:
:exploding_head: You're totally right, let's make those the monitor and the one that watches the monitor!

Reply:
The upboard is named `viam`, which would make this comment confusing. I'll change `pi5canary` on this line to `bogglebean`.

Reply:
could you make this variable name more clear that this variable is for the board watching the watcher and not for the board watching all the board? though I don't have a suggestion so maybe the comment is sufficient.

Reply:
We discussed in person: removing the part about watching the watchmen makes the rest of it make more sense. I was too glib.